charts/kubevirt/0.2.0/values.yaml

```diff
@@ -0,0 +1,7 @@
+kubevirt:
+  image: registry.suse.com/suse/sles/15.5/virt-operator:1.1.0-150500.8.6.1
+  version: 1.1.0-150500.8.6.1
+  pullPolicy: IfNotPresent
+
+kubecli:
+  image: bitnami/kubectl:1.27.3
```

Comment:
Ugh. If this is really needed, maybe we can just use rancher shell? https://github.com/rancher/shell

Reply:
I'll try this one in a follow up once we merge this and v0.2.0 is accessible in Rancher.